libs/SalesforceHybrid/src/com/salesforce/androidsdk/phonegap/ui/SalesforceDroidGapActivity.java

```diff
@@ -116,9 +116,7 @@ public void onCreate(Bundle savedInstanceState) {
     }
 
     protected ClientManager buildClientManager() {
-        return new ClientManager(this, SalesforceSDKManager.getInstance().getAccountType(),
-                SalesforceSDKManager.getInstance().getLoginOptions(),
-                SalesforceSDKManager.getInstance().shouldLogoutWhenTokenRevoked());
+        return SalesforceHybridSDKManager.getInstance().getClientManager();
```

Comment:
Simplified this. We still need `buildClientManager()` because it's overridden by the sub-class in the test to mock a user account.